cloudcredential/v1/codec.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2018 The OpenShift Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"bytes"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+// NewScheme creates a new Scheme
+func NewScheme() (*runtime.Scheme, error) {
```

Comment:
This file is quite unexpected here. Eventual codecs should be created where this is being used, not in API.

Reply:
Sounds like we should move this back to the CCO repo @joelddiaz